lib/liftoff/launchpad.rb

```diff
@@ -11,7 +11,7 @@ def liftoff(options)
         validate_template
         fetch_options
 
-        file_manager.create_project_dir(@config.project_name) do
+        file_manager.create_project_dir(path ? path : @config.project_name) do
```

Comment:
Having this logic here feels like a smell to me. I think it would be best to add a `path` attribute to the project configuration, and put this logic there. The implementation would be fairly trivial:

```ruby
def path
  @path || project_name
end
```

This would also let us add a test around the logic.